src/bulkExport/bulkExportResults.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import AWS from 'aws-sdk';
+
+const EXPIRATION_TIME_SECONDS = 1800;
+const EXPORT_RESULTS_BUCKET = process.env.EXPORT_RESULTS_BUCKET || ' ';
+const EXPORT_RESULTS_SIGNER_ROLE_ARN = process.env.EXPORT_RESULTS_SIGNER_ROLE_ARN || '';
```

Comment:
If you weren't going to use env variables would you just make this a class instead?

Reply:
Either a class or a function with more params like `getBulkExportResults(bucket, role, jobId)`